packages/react-components/react-tree/src/contexts/treeContext.ts

```diff
@@ -19,9 +19,9 @@ export type TreeContextValue = {
 };
 
 export type TreeItemRequest = { itemType: TreeItemType } & (
-  | OmitWithoutExpanding<TreeOpenChangeData, 'open' | 'openItems'>
-  | TreeNavigationData_unstable
-  | OmitWithoutExpanding<TreeCheckedChangeData, 'selectionMode' | 'checkedItems'>
+  | (OmitWithoutExpanding<TreeOpenChangeData, 'open' | 'openItems'> & { requestType: 'open' })
```

Comment:
I can't find any code that actually uses `requestType` , it seems to be passed into requests and lost